lib/spack/docs/docker_for_developers.rst

```diff
@@ -0,0 +1,35 @@
+.. _docker_for_developers:
+
+=================================
+Using Docker for Spack Developers
+=================================
+
+This guide is intended for people who want to use our prepared docker
+environments to work on developing Spack or working on spack packages.  It is
```

Comment:
Are you intentionally making double spaces after `.`s? :)

Reply:
Aye; chalk it up to a habit I learned from grade school.  I should write a quick script to look for these things. [: